Src/Plugins/Tests.Security/Validators/SEC101_005.SlackTokenValidatorTests.cs

```diff
@@ -15,6 +15,9 @@
 
 namespace Microsoft.CodeAnalysis.Sarif.PatternMatcher.Plugins.Security.Validators
 {
+    /// <summary>
+    /// Testing SEC101/005.SlackTokenValidator
```

Comment:
>SlackTokenValidator [](http://example.com/codeflow?start=27&length=19)

The name of this rule is wrong, it should be SlackApiToken. When you fix this, be sure to indicate the deprecated rule name.

Reply:
Oops, I think SlackApiKey is actually the right rule name.